README.md

```diff
@@ -50,6 +50,9 @@ features = ["ron_conf"]
 default-features = false
 ```
 
+### Changing error messages
```

Comment:
We should change this to Heading 2 (`##`) so it doesn't appear to be apart of the feature flags section.

Reply:
That's fair i missed that when i was doing it